### PR TITLE
Add command to remove dropping/dropped collection meta

### PR DESCRIPTION
**Pull request by congqixia**
Add `remove collection-drop` command to clean dropping/dropped collection/channel meta
/kind improvement

### CODE REVIEW

File: states/etcd/remove/collection_dropping.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+	cmd.Flags().Bool("run", false, "flags indicating whether to execute removed command")
+	cmd.Flags().Int64("collectionID", 0, "collection id to remove")
+	return cmd
+}
+
+func cleanCollectionDropMeta(cli clientv3.KV, basePath string, collection *models.Collection) {
+	var ops []clientv3.Op
+	// remove collection meta
+	ops = append(ops, clientv3.OpDelete(collection.Key()))
```
Comment (czs007) [inline states/etcd/remove/collection_dropping.go:67]: how about add logs for every op?

Comment (congqixia) [reply states/etcd/remove/collection_dropping.go:67]: done

File: models/collection.go [hunk 2]
```diff
@@ -32,6 +32,10 @@ type Collection struct {
 	lazyLoad func(*Collection)
 }
 
+func (c *Collection) Key() string {
+	return c.key
```
Comment (czs007) [inline models/collection.go:36]: how about check whether key is nil or "" , if key is nil or "", we should  panic?

Comment (congqixia) [reply models/collection.go:36]: this key is set when collection meta is list. will add check empty logic when use it's value

**Pull request merged**